ui/StatusQ/src/StatusQ/Components/StatusRoundedMedia.qml

```diff
@@ -119,6 +129,24 @@ StatusRoundedComponent {
         visible: !root.isError && !root.isLoading
     }
 
+    MouseArea {
+        anchors.fill: parent
+        propagateComposedEvents: true
```

Comment:
I can't tell what is the reason of enabling `propagateComposedEvents`. I would expect to have side effects of allowing click to propagate to parent while handing it ourself. This way we could have a double handling in different contexts.